### DISCUSSION TITLE
Nepal Country Code in not available for Two Factor Authentication

### DISCUSSION THREAD

**Discussion by TandukarRajan** [community/community#19954] [Other Feature Feedback, Questions, & Ideas]
Nepal country code not found in Drop down for two factor authentication.
![image](https://user-images.githubusercontent.com/89002468/177079242-a8964a1e-19de-4ca5-a3d4-0a99ecbc2664.png)

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
